map2loop/thickness_calculator.py

```diff
@@ -372,7 +393,10 @@ def compute(
                 logger.warning(
                     f"Thickness Calculator InterpolatedStructure: Cannot calculate thickness between {stratigraphic_order[i]} and {stratigraphic_order[i + 1]}\n"
                 )
-
+                
+        self.lines = geopandas.GeoDataFrame(geometry=[line[0] for line in _lines], crs = basal_contacts.crs)
```

Comment:
Change lines to remove the list comprehension

Reply:
https://github.com/Loop3D/map2loop/pull/161/commits/63af48d7d2bbb644a0a004efc2289a07ccb90dff

Reply:
Had to revert this back. 
The lst comprehension was the only way I could find to build the gdf. I do not know of another way to build the gdf - please let me know if you have any other suggestions

Reply:
Fixed in https://github.com/Loop3D/map2loop/pull/161/commits/bf22a328c50d4def89ad78f5e600c9b35256b1ae